### ISSUE TITLE
_arrayIsArray not defined

### ISSUE THREAD

**Issue by msongz** [Klustre/extender#3]
in this case, `_arrayIsArray()` not defined

```js
let str = "Hello";
alert( [...str] ); // H,e,l,l,o
```

**Comment by Klustre** [comment]
This is an issue with [babel-preset-extendscript](https://github.com/fusepilot/babel-preset-extendscript)

**Comment by msongz** [comment]
Hi @Klustre , `_arrayIsArray()` is "exist" at some situations, I can manually change it to `_arrayIsArray2()` so it can works properly, 
but how can we automatic this workaround? 
maybe adding some user defined replacement each builds?

**Comment by Klustre** [comment]
This needs to be fixed upstream in `babel-preset-extendscript` because the issue exists there. I'm a maintainer, but haven't gotten to work on it yet.

Can you open an issue in `babel-preset-extendscript` and show some of the things you've tried and their outcome? Like, when does `_arrayIsArray` exists and when not, etc.

I'm keeping the issue open here as a reference for others until it's fixed.

**Comment by msongz** [comment]
yeah, sure https://github.com/fusepilot/babel-preset-extendscript/issues/14#issue-1437451055
but I have no idea how the _arrayIsArray work or not
